packages/breadcrumb/src/BreadcrumbItem.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { styled } from '@project44-manifest/react-styles';
+
+export const StyledBreadcrumbItem = styled('li', {
+  d: 'flex',
+  alignItems: 'center',
+  color: '$text-secondary',
+
+  '& a': {
+    typography: '$subtext',
+    color: 'inherit',
+    textDecoration: 'none',
+  },
+
+  '&:last-child svg': {
```

Comment:
We probably want to do the same here as above. We should avoid use tags as selectors

Reply:
Done